Automattic/wp-calypso
# TeamCity: Add extension function and refactor ETK build

#### Changes proposed in this Pull Request
- refactors the ETK build into a separate file
- adds an extension function for common bash script properties.
- Adds phpcs to ETK

#### Testing instructions
- Check build output for ETK TC task and make it passes, and that the lint check passes.
